fastjet/internal/delaunay/triangle.go

```diff
@@ -121,6 +122,42 @@ func (t *Triangle) remove() []*Point {
 	return updated
 }
 
+// circumcenter returns the x,y coordinates of the circumcenter of the triangle
+func (t *Triangle) circumcenter() (x, y float64) {
+	m1 := (t.A.x - t.B.x) / (t.B.y - t.A.y)
```

Comment:
there are a few divisions by the same value (modulo the sign).

not sure whether `circumcenter` will end up in the hot path, but perhaps put a `TODO` to store the inverse of these delta values and use that instead.

Reply:
I'm not sure I got it right. 
For example since there is `.../(t.B.y - t.A.y)` and `.../(t.A.y - t.B.y)` we could have `invDeltaAB := 1/(t.B.y - t.A.y)` and use an if statement to check if it is positive or negative.
Did I understand that correctly?

Reply:
leaving as a TODO is fine for now.